glib/src/object.rs

```diff
@@ -1105,7 +1105,7 @@ impl Object {
         unsafe { Object::new_internal(type_, &params) }
     }
 
-    pub fn new_generic(type_: Type, properties: &[(&str, Value)]) -> Result<Object, BoolError> {
+    pub fn with_values(type_: Type, properties: &[(&str, Value)]) -> Result<Object, BoolError> {
```

Comment:
Maybe `with_type_and_values`? Like `with_details_and_values`. I don't like neither though.

Reply:
You need to provide the type in any case, so mentioning it in the function name seems not very useful. Like this should be fine.